Rocket.Chat/Managers/AppManager.swift

```diff
@@ -155,6 +155,7 @@ extension AppManager {
         SocketManager.sharedInstance.connectionHandlers.removeAllObjects()
         SocketManager.disconnect { (_, _) in
             DispatchQueue.main.async {
+                UIApplication.shared.statusBarStyle = .default
```

Comment:
This line was removed recently because it does nothing since we manage the status bar style per view controller